lib/bokken_web.ex

```diff
@@ -17,14 +17,34 @@ defmodule BokkenWeb do
   and import those modules here.
   """
 
-  def controller do
+  def static_paths, do: ~w(images favicon.ico dojo.html robots.txt)
+
+  def controller(version \\ "1.7") do
+    result =
+      case version do
+        "1.7" ->
+          quote do
+            use Phoenix.Controller,
+              namespace: BokkenWeb,
+              formats: [:html, :json],
+              layouts: [html: BokkenWeb.Layouts]
+          end
+
+        _ ->
+          quote do
+            use Phoenix.Controller, namespace: BokkenWeb
+          end
+      end
+
     quote do
-      use Phoenix.Controller, namespace: BokkenWeb
+      unquote(result)
 
       import Plug.Conn
       import BokkenWeb.Gettext
       import Bokken.Guards
       alias BokkenWeb.Router.Helpers, as: Routes
```

Comment:
Maybe this should go to the 1.6 version only 💪

Reply:
Why?

Reply:
From now on, we should use the sigil p for routes 💪

And stop using Routes helper module

Reply:
https://hexdocs.pm/phoenix/routing.html#verified-routes